packages/plugin-browser-device/device.js

```diff
@@ -14,6 +13,7 @@ const getDeviceId = () => {
       return id
     }
 
+    const cuid = require('@bugsnag/cuid')
```

Comment:
Have you checked what this gets compiled to? I think it's fine as all of our requires get flattened (so there's no actual runtime cost of this) but worth double checking that this won't do something unexpected

Reply:
this does indeed get flattened - it just changes the scope of the function here.